spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -1826,6 +1831,11 @@ void dump_spdm_key_exchange_rsp(const void *buffer, size_t buffer_size)
         return;
     }
 
+    /*change global m_req_slot_id when key exchange*/
+    if (spdm_response->req_slot_id_param != 0xF) {
+        m_req_slot_id = spdm_response->req_slot_id_param;
+    }
+
```

Comment:
1) I do not find any rang check (0~7).

2) If slot_id_param is 0xF, then what is `m_req_slot_id` value? Last value ? Always zero? pre-provisioned public cert ? Or always error ? Please add comment to describe it clearly.

Reply:
3) m_req_slot_id might be assigned in ENCAPSULATED_RESPONSE_ACK message . I do not find that code.

Reply:
I have added the check and comments.   And the `m_req_slot_id` can also be changed in `ENCAPSULATED_RESPONSE_ACK` now. Thanks.